compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDistributeCoresAndObjectFifos.cpp

```diff
@@ -506,17 +931,51 @@ void AMDAIEDistributeCoresAndObjectFifosPass::runOnOperation() {
           moduleOp, std::move(unrollLocalLoopsPatterns)))) {
     return signalPassFailure();
   }
+  LLVM_DEBUG(llvm::dbgs() << "Module after AMDAIEUnrollLocalLoops: \n"
+                          << moduleOp << "\n");
+
+  // Insert `amdaie.logicalobjectfifo.access` operations which retrieve the
+  // memrefs from logical objectfifos and update the computational operations to
+  // operate on these local memrefs. These access operations will be used to
+  // assign local AIE tiles to local logical objectFifos later.
+  if (failed(insertLogicalObjectFifoAccess(moduleOp))) {
+    return signalPassFailure();
+  }
```

Comment:
Add `emitOpError` message before `return signalPassFailure()`. Same for the next few failures.

Reply:
Done